docs/api/_index.md

```diff
@@ -0,0 +1,263 @@
+
+# Notification Manager HTTP API
+
+The Notification Manager Deployment exposes an HTTP API for receiving alerts, sending notifications, and verifing config.
+
+> The default port of Notification Manager is `19093`.
+
+- [`Receive alerts`](#Receive-alerts)
+- [`Send notifications`](#Send-notifications)
+- [`Verify`](#Verify)
+
+## Receive alerts
+
+> Post /api/v2/alerts
+
+This API used to send alerts to Notification manager. Notification manager will receive alerts after this API called, and then send notifications to the receivers which defined in the cluster. 
```

Comment:
=>
This API is used to send alerts to Notification Manager. Notification Manager will then send notifications to the receivers defined in the cluster.